apollo-router/src/query_planner/fetch.rs

```diff
@@ -611,8 +611,8 @@ impl FetchNode {
         &mut self,
         subgraph_schemas: &SubgraphSchemas,
         supergraph_schema_hash: &str,
-    ) {
-        let doc = self.parsed_operation(subgraph_schemas);
+    ) -> Result<(), ValidationErrors> {
+        let doc = self.parsed_operation(subgraph_schemas)?;
```

Comment:
note that this will likely conflict with https://github.com/apollographql/router/pull/5047/files

Reply:
I merged dev into this branch and didn’t find a conflict